packages/engine/lib/execution/src/worker_pool/mod.rs

```diff
@@ -148,8 +148,10 @@ impl WorkerPool {
 
         let futs = workers
             .into_iter()
-            .map(|mut c| {
-                tokio::spawn(async move { c.run().await.map_err(Error::from) }.in_current_span())
+            .map(|mut worker| {
```

Comment:
Thank you 👏 

Honestly might be worth us enabling the no one char variable names lint and just spending a few hours removing all these damn things.

Not worth the time right now though.